customization-base/src/main/java/com/azure/autorest/customization/implementation/ls/EclipseLanguageClient.java

```diff
@@ -215,14 +209,7 @@ public void close() {
     }
 
     private static <T> T sendRequest(Connection connection, String method, Object param, Type responseType) {
-        try {
-            String response = OBJECT_MAPPER.writeValueAsString(connection.requestWithSerializedObject(
-                OBJECT_MAPPER.constructType(JsonNode.class), method, GSON.toJson(param)));
-
-            return GSON.fromJson(response, responseType);
-        } catch (IOException e) {
-            throw new UncheckedIOException(e);
-        }
+        return GSON.fromJson(connection.requestWithSerializedObject(method, GSON.toJson(param)), responseType);
```

Comment:
Are we replacing it wit gson?

Reply:
Why do we want to use GSON here?

Reply:
The types used by the Eclipse dependencies use GSON

Reply:
It doesn't have to be done in this PR. Please file an issue and resolve this comment.